src/components/OnboardingDialog/index.stories.js

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from 'react'
+import { Button } from '../Button'
+import { H1, H3 } from '../Heading'
+import { Input } from '../Input'
+import styled from '@emotion/styled'
+import { space } from '../../theme'
+import { Text } from '../Text'
+import {
+  OnboardingDialog,
+  OnboardingDialogBody,
+  OnboardingWrapper,
+  useOnboarding,
+} from './index'
+import { ContentTransition } from '../ContentTransition'
+
+const Body = styled.div`
+  padding: ${space[32]};
+`
+
+const Form = styled.form`
+  display: grid;
+  grid-gap: ${space[16]};
+  margin: ${space[32]} 0;
+`
+
+const Actions = styled.div`
+  display: grid;
+  margin-top: ${space[16]};
+  grid-template-columns: 200px 200px;
+  grid-gap: ${space[8]};
+`
+
+const StorybookDialog = () => {
+  const { show, hide, getOnboardingProps, state, active } = useOnboarding({
+    defaultTransitionState: 'entered',
```

Comment:
Is the prop `defaultTransitionState` really needed, could you explain what it actually does?

Reply:
In the current implementation the default state is hard coded with 'exited'. This works fine with our default dialogs, but with this specific dialog it will clearly show that the backdrop will fade in a bit after the page load which looks awful.